### DISCUSSION TITLE
DIAMOND v2.0.13

### DISCUSSION THREAD

**Discussion by bbuchfink** [bbuchfink/diamond#522] [General]
- Fixed a bug that caused invalid bit scores in frameshift alignment mode.

<hr /><em>This discussion was created from the release <a href='https://github.com/bbuchfink/diamond/releases/tag/v2.0.13'>DIAMOND v2.0.13</a>.</em>

**Comment by mpovidlov1** [comment]
For some reason, the latest version (v2.0.13, as well as v2.0.12) crashes on my ubuntu 16.04.4 (Xenial Xerus):
ubuntu@P4:~$ diamond
Illegal instruction (core dumped)

The version 2.0.11 worked fine on this machine.

At the same time, the latest version does work on ubuntu 18.04.3 LTS (Bionic Beaver).

I wonder if I am alone or are there similar reports?

Thanks

**Reply by mpovidlov1** [reply]
Update: another machine running ubuntu 16 has the same problem.

**Reply by bbuchfink** [reply]
> For some reason, the latest version (v2.0.13, as well as v2.0.12) crashes on my ubuntu 16.04.4 (Xenial Xerus): ubuntu@P4:~$ diamond Illegal instruction (core dumped)
> 
> The version 2.0.11 worked fine on this machine.
> 
> At the same time, the latest version does work on ubuntu 18.04.3 LTS (Bionic Beaver).
> 
> I wonder if I am alone or are there similar reports?
> 
> Thanks

I can confirm this error and will work on a fix. For now you can try to use the conda version or compile from source.

**Reply by bbuchfink** [reply]
Update: This fix is needed to compile a working version for old platforms: c35483caf7db2ebca76a1087a55ff6caaad50a3d
